Jonathan.md

```diff
@@ -0,0 +1,3 @@
+Jonthan 
```

Comment:
The commit says "Jonathan" and the file says "Jonathan" but then the text says "Jonthan" (note the missing 'a' between 'n' and 't')

Please decide which spelling you'd like to use and use it consistently across your changes, thank you! ✨

Reply:
i just missed spelled my name it has the a